_posts/fr/newsletters/2023-08-09-newsletter.md

```diff
@@ -0,0 +1,303 @@
+  avant d'être fusionnée s'il s'agit simplement d'une refonte. D'autres projets qui rencontrent des difficultés pour faire
+  réviser des refontes ou des petits changements selon les mêmes normes élevées qu'ils appliquent aux autres demandes de pull
+  pourraient vouloir examiner la nouvelle politique de Rust Bitcoin.
+
+- [BOLTs #759][] ajoute la prise en charge des [messages en oignon][topic onion messages] à la spécification LN. Les messages
+  en oignon permettent d'envoyer des messages unidirectionnels à travers le réseau. Comme les paiements (HTLC), les messages
+  utilisent le chiffrement en oignon de sorte que chaque nœud de transfert ne sait que quel pair lui a envoyé le message et quel
+  pair devrait recevoir le message ensuite. Les charges utiles des messages sont également chiffrées de sorte que seul le
+  destinataire final puisse les lire. Contrairement aux HTLC transférés, qui sont bidirectionnels---les flux d'engagement
+  descendent vers le destinataire et la préimage nécessaire pour réclamer le paiement remonte vers le dépensier---la nature
+  unidirectionnelle des messages en oignon signifie que les nœuds de transfert n'ont pas besoin de stocker quoi que ce soit à
+  leur sujet après que le message a été transféré, bien que certains mécanismes de protection contre les attaques par déni de
+  service proposés dépendent de la conservation d'une petite quantité d'informations agrégées sur une base par pair (voir la
+  [Newsletter #207][news207 onion]). La messagerie bidirectionnelle peut être réalisée en incluant un chemin de retour dans le
+  message initial. Les messages en oignon utilisent les [chemins aveugles][topic rv routing], qui ont été ajoutés à la
+  spécification LN il y a quelques mois (voir [Newsletter #245][news245 blinded]), et les messages en oignon sont eux-mêmes
+  utilisés par le protocole [offers][topic offers] en cours de développement.
+
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="27746,6376,6475,6466,6473,6253,5675,863,1945,759,28132,28130" %}
+[news245 blinded]: /fr/newsletters/2023/04/05/#bolts-765
+[towns dos]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-July/004020.html
+[news86 hold fees]: /en/newsletters/2020/02/26/#reverse-up-front-payments
+[shikhelman dos]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-July/004033.html
+[towns dos2]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-August/004035.html
+[kcs endorsement]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-August/004034.html
+[todd rbf]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2023-July/021823.html
+[towns rbf]: https://github.com/bitcoin/bitcoin/pull/28132#issuecomment-1657669845
+[news207 onion]: /en/newsletters/2022/07/06/#onion-message-rate-limiting
+[news261 jamming]: /en/newsletters/2023/07/26/#channel-jamming-mitigation-proposals
```

Comment:
I think:

```suggestion
[news261 jamming]: /fr/newsletters/2023/07/26/#propositions-d-attenuation-des-attaques-de-saturation-des-canaux
```

Reply:
@Copinmalin I applied this suggestion to the code. Let me know if you think this PR is good to go